### ISSUE TITLE
[a11y] missing meaningful label of sort order indicator

### ISSUE THREAD

**Issue by susnux** [nextcloud/groupfolders#4463]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. open admin settings
2. see sort order indicator accessibility information

### Expected behaviour
Should have a proper label like "sorted ascending"

### Actual behaviour
Just the icon is rendered

<img width="1755" height="647" alt="Image" src="https://github.com/user-attachments/assets/5040dae0-4558-4a59-b90a-16fda2ec9f90" />
